test/fixtures/magazines.yml

```diff
@@ -0,0 +1,7 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
+  title: MyString
+
+two:
+  title: MyString
```

Comment:
Small tip on fixtures - these are going to be used in your tests, so having both `:one` and `:two` have the same title attribute might make it hard to confirm behavior.

If you feel it's helpful, you mind consider making these unique values